### DISCUSSION TITLE
ETHEREUM_JSONRPC_VARIANT=besu Error

### DISCUSSION THREAD

**Discussion by gmba0212** [blockscout/blockscout#8973] [Q&A]
I want to connect BlockScout and Hyperledger Besu node, but {"time":"2023-12-11T06:52:46.169Z","severity":"error","message":"failed to fetch internal transactions for blocks: ** (ErlangError) Erlang error: [%{code: -32604, data: %{\"blockNumber\" => 1342663}, message: \"Method not enabled\"}, %{code: -32604, data: %{ \"blockNumber\" => 1342664}, message: \"Method not enabled\"} The error continues to occur and does not start properly.

**Comment by gmba0212** [comment]
ENV settings were entered normally.      ~/blockscout/docker-compose# docker exec -it f74 env | grep besu
ETHEREUM_JSONRPC_VARIANT=besu

**Comment by sl1depengwyn** [comment]
How do you run your node? You need to add `TRACE` to `--rpc-http-api=` option

**Reply by gmba0212** [reply]
Thank you. I was worried about this issue for several days, but it was resolved in one go. !!!
